### ISSUE TITLE
Game breaking bug in cool cool mountain

### ISSUE THREAD

**Issue by Peppe021203** [Fluvian/tpp#12]
Just stumbled upon this, idk if it happens on an emulator but i do know that it happens on console. Basically at the bottom part of cool cool mountain there is a section where you need to jump on ice blocks, when mario reaches the second block his model gets messed and on the third it jitters a lot and the game crashes

**Comment by Fluvian** [comment]
Are you using a flash cart? Make sure the connectors to the console are clean and that the SD Card is not corrupted/connectors are clean.

**Comment by Peppe021203** [comment]
yeah i'm using a flash cart, the connectors have been cleaned an the sd card is a brand new one, still happens

**Comment by iProgramMC** [comment]
Are you able to send a recording?
